hibernate-envers/src/main/java/org/hibernate/envers/Audited.java

```diff
@@ -37,6 +39,25 @@
 	 */
 	RelationTargetAuditMode targetAuditMode() default RelationTargetAuditMode.AUDITED;
 
+	/**
+	 * Specifies if the entity that is the relation target isn't found, how should the system react.
+	 *
+	 * The default is to use the behavior configured based on the system property:
+	 * {@link org.hibernate.envers.configuration.EnversSettings#GLOBAL_RELATION_NOT_FOUND_LEGACY_FLAG}.
+	 *
+	 * When the configuration property is {@code true}, this is to use the legacy behavior which
+	 * implies that the system should throw the {@code EntityNotFoundException} errors unless
+	 * the user has explicitly specified the value {@link RelationTargetNotFoundAction#IGNORE}.
+	 *
+	 * When the configuration property is {@code false}, this is to use the new behavior which
+	 * implies that the system should ignore the {@code EntityNotFoundException} errors unless
+	 * the user has explicitly specified the value {@link RelationTargetNotFoundAction#ERROR}.
+	 *
+	 * @since 6.0
```

Comment:
`6.0` is not the correct version. I would like to remove the above, rather than correcting it, since it will need to be corrected again if/when backported to 5.3 branch.

Reply:
@gbadner That seems reasonable.  I completely missed that when I ported the code to master.